### ISSUE TITLE
[Feature request] Rearrange/Remove UI hierarchy after creation.

### ISSUE THREAD

**Issue by repalash** [cocopon/tweakpane#218]
Hello, firstly, this is truly an amazing library, please keep it up.

Now, I am trying to create a hierarchy list (like a tree view) using nested folders but couldn't find any proper way to remove, reorder or insert(at a random index) elements in a folder or the root.  
Some interface in the folder and root API to insert components at a position and remove an element would be very helpful. Along with some way to traverse the folder hierarchy or list children elements.

**Comment by cocopon** [comment]
Thank you for using Tweakpane.

Currently disposing a component and re-creating it is the only way to do so.

```js
const input = folder.addInput(...);

// Remove the input from the folder
input.dispose();

// Insert the component to the specified position
const newInput = folder.addInput(..., {
  index: 2,
});
```

But if Tweakpane is going to be a UI library rather than just a tweaking pane (#180), it should be supported.

**Comment by cocopon** [comment]
Implemented in #251. Please wait for the next release.
